nss-crystal-cobras/WorkforceManagement
# Ac department details

Closes Ticket #7 

User can see department details; details page has name of dept as header and shows list of employees assigned to that department.

## Type of change


# Testing Instructions for Change Made

git fetch --all
git checkout ac-department-details
run BangazonWorkforce
in nav bar: Departments --> Details
Details page should have department name as a header, followed by list of employees within dept


# Checklist:
